src/components/AppFooter.vue

```diff
@@ -0,0 +1,101 @@
+<script>
+import SOLogo from './SOLogo.vue'
+import {library} from '@fortawesome/fontawesome-svg-core'
+import {FontAwesomeIcon} from '@fortawesome/vue-fontawesome'
+import {faFacebook, faInstagram, faLinkedin, faTwitter} from '@fortawesome/free-brands-svg-icons'
+
+library.add(faInstagram, faFacebook, faLinkedin)
+
+export default {
+  components: {
+    SOLogo, FontAwesomeIcon
+  },
+  data() {
+    return {
+      footer_menus: [
+        {
+          title: 'Contact',
+          items: ['Link 1', 'Link 2', 'Link 3']
+        },
+        {
+          title: 'Contact',
+          items: ['Link 1', 'Link 2', 'Link 3']
+        }
+      ],
+      social_media_profiles: [
+        {
+          profile: 'instagram',
+          icon: faInstagram,
+          link: ''
+        },
+        {
+          profile: 'facebook',
+          icon: faFacebook,
+          link: ''
+        },
+        {
+          profile: 'linkedin',
+          icon: faLinkedin,
+          link: ''
+        },
+        {
+          profile: 'twitter',
+          icon: faTwitter,
+          link: ''
+        }
+      ]
+    }
+  }
+}
+</script>
+
+
+<template>
+  <div class="container w-100">
+    <div class="row">
+      <div class="col-12 col-md-8 d-flex align-items-center
+      justify-content-center justify-content-md-start my-md-0 my-4">
+        <SOLogo/>
+      </div>
+      <div class="col-12 col-md-4">
+        <div class="row justify-content-end">
+          <div v-for="menu in footer_menus" class="col-md-4 col-12 text-center text-md-start">
+            <div class="footer-menu-title mt-2 mb-1">{{ menu.title }}</div>
+            <div v-for="item in menu.items">
+              {{ item }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <hr>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-4 col-12 d-flex justify-content-center justify-content-md-start
+      align-items-center order-last order-md-first my-2 my-lg-0">
+        <span>© Speech Olympiad 2022</span>
+      </div>
+      <div class="col-md-8 col-12 order-first order-md-last">
+        <div class="row">
+          <div class="col-12 col-md-6 my-2 my-md-0
+          d-flex justify-content-center justify-content-md-end align-items-center"><span>Follow us on</span></div>
+          <div v-for="social_media in social_media_profiles" class="col my-2 my-md-0">
```

Comment:
could we center the individual icons horizontally in their cells, (For small screen sizes it looks asymmetrical)